CONTRIBUTING.md

```diff
@@ -14,6 +14,9 @@ source .venv/bin/activate
 python -m pip install -e '.[dev]'
 ```
 
+> [!TIP]
+> Add an extra name for each database you want to use (e.g. `[dev,mysql]`,  `[dev,postgres]` or `[dev,mysql,postgres]`).
```

Comment:
Suggestion: Be more explicit that installing these extras is optional
```suggestion
> Add an extra name for each database you want to develop with (e.g. `[dev,mysql]`,  `[dev,postgres]` or `[dev,mysql,postgres]`). This is optional.
```

Reply:
Thanks, I'll update the PR.